src/shared/error-messages.ts

```diff
@@ -5,6 +5,7 @@ export const ERROR_MESSAGES = {
 	},
 	CONTACT: {
 		INVALID_SINGAPORE_NUMBER: "Invalid Singapore number",
+		INVALID_FIXED_COUNTRY: (country: string) => `Invalid ${country} number`,
```

Comment:
no need error message for invalid country, let's infer from the existing SG / international number error messages